src/api/routes/faucets.ts

```diff
@@ -69,6 +69,15 @@ export function createFaucetRouter(db: PgWriteStore): express.Router {
   const router = express.Router();
   router.use(express.urlencoded({ extended: true }));
   router.use(express.json());
+  const missingBtcConfigMiddleware = (
+    err: Error,
+    req: express.Request,
+    res: express.Response,
+    next: express.NextFunction
+  ) =>
+    err.message === 'BTC Faucet not fully configured.'
```

Comment:
This error message string comparison is untyped and seems brittle. If using this pattern, then an specific Error class for this should be created. The only time we do error message string comparisons is when dealing with errors from other dependencies that we don't control.

Reply:
Enum or constant can also be used to avoid untyped string comparison. 
 const FaucetError = {
  BTC_FAUCET_NOT_CONFIGURED: 'BTC Faucet not fully configured.'
}

Comparison:
 err.message === FaucetError.BTC_FAUCET_NOT_CONFIGURED 
 
 @zone117x what are your thoughts on this?

Reply:
That approach seems a bit strange. I'd just go with the existing pattern of creating an Error type and checking for it. Example:
https://github.com/hirosystems/stacks-blockchain-api/blob/6d66c3ccc1b270b70edf4919efde40b582c2cbd2/src/api/init.ts#L300